xdsl/dialects/mod_arith.py

```diff
@@ -0,0 +1,40 @@
+from abc import ABC
+
+from xdsl.dialects.arith import signlessIntegerLike
+from xdsl.ir import Attribute, Dialect, SSAValue
+from xdsl.irdl import IRDLOperation, irdl_op_definition, operand_def, result_def
+from xdsl.traits import Pure
+
+
+class ModArithOp(IRDLOperation, ABC):
+    # assemblyFormat = "operands attr-dict `:` functional-type(operands, results)"
+    pass
+
+
+class BinaryOp(ModArithOp):
```

Comment:
```suggestion

class BinaryOp(ModArithOp):
```

Reply:
Not sure we want double-newlines, but I guess others know the coding style of xDSL better.

Reply:
This is entirely handled by Ruff :)